src/SmearedJetProducer.cc

```diff
@@ -184,7 +186,7 @@ double SmearedJetProducer::getSmearFactor(TLorentzVector jet, bigTree& theBigTre
 TLorentzVector SmearedJetProducer::getSmearedJet(TLorentzVector jet, bigTree & theBigTree)
 {
   // Get the smear factor
-  double smearFactor = SmearedJetProducer::getSmearFactor(jet, theBigTree);
+  double smearFactor = SmearedJetProducer::getSmearFactor(jet, theBigTree, true);
```

Comment:
are we using `getSmearedJet` anywhere? and if so is it ok to always set this to `true`?